src/eyeTracker.m

```diff
@@ -218,52 +166,54 @@
                     return
                 end
 
-                % record a few samples before we actually start displaying
-                % otherwise you may lose a few msec of data
+                % Record a few samples before we actually start displaying otherwise you may lose
+                %  a few msec of data.
                 WaitSecs(0.1);
 
-                % HERE START THE STIMULATION OF THE BLOCK
-                % to mark the beginning of the trial
+                % Mark the beginning of the trial, here start the stimulation of the experiment.
                 Eyelink('Message', 'SYNCTIME');
```

Comment:
I would change the message to "start_recording" and would keep the "SYNC_TIME" for the beginning of the experiment.

This is important if we are starting and stopping the eyetracker several times in one run.